include/rocksdb/listener.h

```diff
@@ -206,6 +230,22 @@ struct FileOperationInfo {
   }
 };
 
+struct BlobFileInfo {
+  std::string blob_file_path;
+  uint64_t blob_file_number;
+  uint64_t total_blob_count;
+  uint64_t total_blob_bytes;
+  CompressionType compression_type;
+};
+
+struct BlobFileGarbageInfo {
+  std::string blob_file_path;
+  uint64_t blob_file_number;
+  uint64_t garbage_blob_count;
+  uint64_t garbage_blob_bytes;
+  CompressionType compression_type;
+};
```

Comment:
Why two structs with identical content?

Reply:
They're not identical: one has `total_blob_{count,bytes}` and the other has `garbage_blob_{count,bytes}`.

Reply:
The contents are exactly the same but the variable name is different.  And you are putting the values in two separate lists that are already differentiated.  This means that to traverse the two lists I need to write twice as much code as I cannot process them using a common subroutine.

Reply:
To me, this is a feature not a bug. The two structures (and the blob count/bytes variables) have different semantics; using dedicated structures improves type safety and is more future-proof.